docker/run_container

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env bash
```

Comment:
This script should not be in the public repo.

Reply:
Removed run_container in 2788606c0f0fc5f6431a1eca9f06780015e52c0e